scripts/locations_data_load/locations_lambda.py

```diff
@@ -0,0 +1,283 @@
+#!/usr/bin/env python3
+
+import requests
+
+import pandas as pd
+import boto3
+import uuid
+import datetime
+from boto3.dynamodb.conditions import Attr
+
+
+def lambda_handler(event, context):
+    print("Fetching organizations data.")
+    fetch_organizations()
+    print("Fetching Y organizations data.")
+    fetch_Y_organizations()
+
+
+# Get parameters from store
+def get_ssm(id, secret):
+    ssm = boto3.client("ssm")
+    client_id = ssm.get_parameter(Name=id)["Parameter"]["Value"]
+    client_secret = ssm.get_parameter(Name=secret, WithDecryption=True)["Parameter"][
+        "Value"
+    ]
+    return client_id, client_secret
+
+
+def get_api_token():
```

Comment:
May be wrong but don;t think this is covered by unit test which means we have no way of testing that   data is build correctly. Headers is all hardcoded so not such an issue.
One option could be to split into two functions - one that returns data and header and can be unit tested ; one that issues request and returns token. Don't think we can unit test the request (that is an integration test I suppose) but we could unit test the rest